spec.html

```diff
@@ -332,6 +332,21 @@ <h1>Number object</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-terms-and-definitions-bigint-value">
+      <h1>BigInt value</h1>
```

Comment:
This placement separates the Number type from the various instances of the Number type (like Infinity). I think it would be better to place this after the definition of NaN.